src/style.css

```diff
@@ -1,3 +1,82 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-align: left;
   margin: 0;
 }
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  max-width: 1200px;
+  position: relative;
+  box-sizing: border-box;
+}
+
+.nav_list {
+  display: flex;
+  align-items: stretch;
+  justify-content: space-around;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  height: 60px;
+  width: 100%;
+}
+
+.nav_item {
+  margin: 0 10px;
+  position: relative;
+}
+
+.nav_item a {
+  display: flex;
+  align-items: center;
+  color: black;
+  text-decoration: none;
+  position: relative;
+  height: 100%;
+}
+
+.nav_item a.is-active {
+  color: #00acdc;
+}
+
+.nav_item a.is-active::after {
+  content: '';
+  position: absolute;
+  background-color: #00acdc;
+  width: 100%;
+  height: 4px;
+  left: 0;
+  border-radius: 8px;
+  bottom: 0;
+  box-sizing: border-box;
+}
+
+.nav_item a:hover {
+  color: #00acdc;
+  border-bottom: 2px solid #00acdc;
+}
+
+[data-qa='hover'] {
```

Comment:
The selector `[data-qa='hover']` is used to apply styles, which is fine, but ensure that this attribute is consistently used in the HTML for elements that require this styling. If it's only used once, consider simplifying the selector or ensuring it's necessary.